airbyte-integrations/connectors/source-primetric/source_primetric/schemas/assignments.json

```diff
@@ -339,6 +381,242 @@
         "null",
         "boolean"
       ]
+    },
+    "financial_budget_mode" : {
+      "type" : [
+        "null",
+        "integer"
+      ]
+    },
+    "financial_client_currency" : {
+      "type" : [
+        "null",
+        "string"
+      ]
+    },
+    "financial_client_currency_exchange_rate" : {
+      "type" : [
+        "null",
+        "number"
+      ]
+    },
+    "financial_total_scheduled_income" : {
+      "type" : [
+        "null",
+        "object"
+      ],
+      "properties" : {
```

Comment:
Extracting this as for the `custom_attributes.json` would be really nice 😄

Reply:
Done!